src/pages/gettingstarted/designers.hbs

```diff
@@ -5,7 +5,77 @@ order: 1
 <div class=" drizzle-o-Layout__section-header">
   <h1 class="drizzle-b-h1">Getting Started - Designers</h1>
 </div>
+<div class="drizzle-o-Layout-content drizzle-b-LongFormText">
+  <div class="drizzle-o-ContentGrouping">
+    <div class="drizzle-c-Alert">
+      <div class="drizzle-c-Alert__icon">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
+          <path d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/>
+        </svg>
+      </div>
+      <p class="drizzle-c-Alert__message sprk-u-mbn">
+        Spark is currently in beta. Please reach out to the
+        <a class="drizzle-b-Link drizzle-b-Link--inline" href="mailto:sparkdesignsystem@quickenloans.com">Spark team</a>
+        for guidance.
+      </p>
+    </div>
+  </div>
 
-<div class=" drizzle-o-Layout-content">
-  <p class="drizzle-b-P">Content coming soon.</p>
+  <div class="drizzle-o-ContentGrouping">
+    <h2 class="drizzle-b-h2">Welcome</h2>
+    <p>
+      This getting started guide is intended for designers. See below for instructions on how to install the design toolkit
+      and start using it to design components.
+    </p>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+    <h2 class="drizzle-b-h2">Spark Design System Toolkit</h2>
+    <p>The Spark Design System is an ever evolving framework that contains all visual assets used to create a
+      responsive web experience. For a designer, Spark provides visual documentation and recommendations on how to
+      apply the visual styles and how to use the components.</p>
+    <p>Every design element housed within Spark has been built out in our Spark Toolkit to help with rapid design and
+      layout prototyping.</p>
+    <p>The Spark Design System Team makes frequent updates to the Spark Toolkit to ensure that it is always up to date
+      with our latest design recommendations.</p>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+    <h2 class="drizzle-b-h2">Sketch</h2>
+    <p>To use the Spark Design System Sketch template, follow these steps:</p>
+    <ul class="drizzle-b-List drizzle-b-List--nested">
+      <li class="sprk-u-mvm">download the <a
+        href="https://github.com/sparkdesignsystem/spark-design-toolkit/raw/master/resources/Spark-Design-Toolkit-Sketch.sketch"
+        class="drizzle-b-Link">sketch template file</a>.
+      </li>
+      <li class="sprk-u-mvm">open file in sketch, then click File -> Save as Template.</li>
+      <li class="sprk-u-mvm">create a new document, by clicking File -> New From Template -> Spark Design
```

Comment:
I think create needs to be capitalized and then a period needs to be added at the end to make it follow the others, or if you wanted to not capitalize things, you would have to remove all the periods